docs/sql/statements/create-role.rst

```diff
@@ -0,0 +1,31 @@
+.. highlight:: psql
+.. _ref-create-role:
+
+===============
+``CREATE ROLE``
+===============
+
+Create a new database role.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+.. code-block:: psql
+
+  CREATE ROLE roleName
+
+Description
+===========
+
+``CREATE ROLE`` is a management statement to create a new database role in the
+CrateDB cluster. The newly created role does not have any special privileges,
```

Comment:
again, since we don't have a way currently to make a role, a user (that can login), I'm hesitant to add this duality. I think we should do it once we enable to change the `login` flag.